ts/sagas/startup/watchLoadMessagesSaga.ts

```diff
@@ -111,6 +111,7 @@ function* loadMessages(
           const cached = cachedMessagesById[message.id];
           return (
             cached === undefined ||
+            cached.message === undefined ||
```

Comment:
I put it there because I was experiencing a runtime exception.
Agree: not the best solution 😄 

I've rolled back most of the changes in this file https://github.com/pagopa/io-app/pull/3441/files#diff-80fa77376f54280f568be0070327a9102639a9b6a6e329a9a88af3ebe8326151